### DISCUSSION TITLE
supporting "releaseToAllUsers"

### DISCUSSION THREAD

**Discussion by iliao58** [fastlane/fastlane#16961] [Ideas]
So far Spaceship::ConnectAPI supports manual app release. There is still a gap in between for the case of "releaseToAllUsers". 
I have done a brief source lookup, and it seems like we can patch the phasedRleaseState for accomplishing this functionality. 

```
releaseToAllUsers: function() {
    return function(e, t) {
        var n = t(),
            r = Object(y.o)(n.storeVersions);
        e(Object(_.a)(g.nc, {
            providerOps: {
                versionId: r
            },
            attributes: {
                phasedReleaseState: v.b
            }
        }))
    }
}
```

It would be amazing if Spaceship::ConnectAPI can have the same parity. Thanks so much!

**Comment by iliao58** [comment]
we can leverage the apple api here. https://developer.apple.com/documentation/appstoreconnectapi/app_store_version_phased_releases

**Comment by kambala-decapitator** [comment]
> So far Spaceship::ConnectAPI supports manual app release

@iliao58 could you share how? I've opened #16994 about that.

**Comment by iliao58** [comment]
@joshdholtz are we planning to add this support? If so, can you please share the ETA? Thanks.
